### PR TITLE
Update vendor-portal-creating-account.md

### PR DESCRIPTION
**Pull request by JustinN-Replicated**
adding auto-join

### CODE REVIEW

**Comment by JustinN-Replicated** [comment]
sc-58440

**Comment by JustinN-Replicated** [comment]
sc-58440

File: docs/vendor/vendor-portal-creating-account.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ To create a vendor account:
   
     - Copy and paste the activation code into the text box and click **Activate**. Your account is now activated.
 
+      :::note
+      Once your account is activated, you may have the option to join a team that you have an existing invite for or to [Auto-Join](/vendor/team-management#enable-users-to-auto-join-your-team) an existing team.
```
Comment (markpundsack) [inline docs/vendor/vendor-portal-creating-account.md:23]:
```suggestion
      Once your account is activated, you may have the option to accept a pending invite, or to [Auto-Join](/vendor/team-management#enable-users-to-auto-join-your-team) an existing team.
```

**Review by markpundsack — APPROVED** [review]

**Comment by paigecalvert** [comment]
Hello! Just a reminder to make sure that the docs team has a chance to review before merging 🙂 @JustinN-Replicated

**Pull request merged**